Tests/SDFCoreTests/MacroInitializerSorterTests.swift

```diff
@@ -51,6 +51,22 @@ final class MacroInitializerSorterTests {
     """
     try await #expect(sut.run(onContent: content.rawValue).contains(expected))
   }
+  
```

Comment:
This is completely unnecessary for this issue, but it's how I discovered this was failing.

Basically I was wondering if the changes introduced in #4 would work still when the nested type contains the macro too, and I was validating with this unit test.

As expected, things work! So nothing to see here, but I think this unit test is still valuable so I decided to commit it anyways.

Reply:
SafeDI does not allow nested macros today, but certainly doesn't hurt to support this potential future case.